SQL/proc_descriptions/procedure_compute_speeds_from_neigborhood_segments.sql

```diff
@@ -9,131 +10,175 @@ DECLARE
-RAISE NOTICE '% segments with assigned speed  found in target area', assigned_segments_count;
-
-CREATE VIEW assigned_segments_in_target_area AS
-	SELECT count(1)
-		FROM nodes_ways_speeds
-				 JOIN nodes_ways from_nodes_ways ON nodes_ways_speeds.from_node_ways_id = from_nodes_ways.id
-				 JOIN nodes_ways to_node_ways ON to_node_ways_id = to_node_ways.id
-				 JOIN target_ways ON from_nodes_ways.way_id = target_ways.id AND to_node_ways.way_id = target_ways.id;
-
-RAISE NOTICE 'computing speed for segments using speed segments within 10 m distance';
-INSERT INTO nodes_ways_speeds
-SELECT
-    from_id, speed, st_dev, to_id, 3 AS quality, count
-FROM node_segments
- JOIN LATERAL (
-	SELECT
-	    avg(speed) AS speed,
-		avg(st_dev) AS st_dev,
-		count(1) AS count
-	FROM speed_segment_data
-	WHERE st_intersects(st_buffer(node_segments.geom, 10), speed_segment_data.geom)
-) computed_speed_small_neighborhood ON TRUE
-WHERE speed IS NOT NULL;
-new_assigned_segments_count = (SELECT count FROM assigned_segments_in_target_area);
-RAISE NOTICE 'speed from close neighborhood computed for % segments', new_assigned_segments_count - assigned_segments_count;
-assigned_segments_count = new_assigned_segments_count;
-
-RAISE NOTICE 'computing speed for segments using speed segments within 200 m distance';
-REFRESH MATERIALIZED VIEW node_segments;
-INSERT INTO nodes_ways_speeds
-SELECT
-	from_id, speed, st_dev, to_id, 4 AS quality, count
-	FROM node_segments
-			 JOIN LATERAL (
-			SELECT
-				avg(speed) AS speed,
-				avg(st_dev) AS st_dev,
-				count(1) AS count
-				FROM speed_segment_data
-				WHERE st_intersects(st_buffer(node_segments.geom, 200), speed_segment_data.geom)
-			) computed_speed_small_neighborhood ON TRUE
-	WHERE speed IS NOT NULL;
-new_assigned_segments_count = (SELECT count FROM assigned_segments_in_target_area);
-RAISE NOTICE 'speed from distant neighborhood computed for % segments', new_assigned_segments_count - assigned_segments_count;
-assigned_segments_count = new_assigned_segments_count;
-
-RAISE NOTICE 'computing speed for remaining segments using average speed';
-REFRESH MATERIALIZED VIEW node_segments;
-WITH average_speed AS (
-	SELECT
-	    AVG(speed) AS average_speed,
-		AVG(st_dev) AS average_st_dev,
-		count(1) AS count
-	FROM speed_segment_data
-)
-INSERT INTO nodes_ways_speeds
-SELECT
-	from_id, average_speed, average_st_dev, to_id, 5 AS quality, count
-	FROM node_segments
-	JOIN average_speed ON TRUE;
-new_assigned_segments_count = (SELECT count FROM assigned_segments_in_target_area);
-RAISE NOTICE 'Average speed assigned to % segments', new_assigned_segments_count - assigned_segments_count;
-
-
-DROP TABLE IF EXISTS speed_segment_data;
-DROP MATERIALIZED VIEW IF EXISTS node_segments;
-DROP VIEW IF EXISTS assigned_segments_in_target_area;
-DROP MATERIALIZED VIEW IF EXISTS target_ways;
+    -- 1. Select the target ways withing the specified area
+    RAISE NOTICE 'selecting target ways';
+
```

Comment:
You are replacing tabs with spaces, which is not good in general, and as a side effect, I see all lines as changed. Please set up your editor accordingly.

Reply:
yeah, you're right. I've using neovim editor and it was set up in such a way, so that I could navigate in those spaces (where tabulation is supposed to be) better. But I guess I could on the final commit replace all 2-spaces with tab character.